main.nf

```diff
@@ -2597,18 +2597,11 @@ process Ascat {
     gender = genderMap[idPatient]
     ascat_purity=params.ascat_purity
     ascat_ploidy=params.ascat_ploidy
-    if (params.ascat_purity && params.ascat_ploidy)
+    purity_ploidy = (params.ascat_purity && params.ascat_ploidy) ? "--purity ${params.ascat_purity} --ploidy ${params.ascat_ploidy}" : ""
```

Comment:
This throws an error for me as `params.ascat_purity` and `params.ascat_ploidy` are not boolean

Reply:
```Error executing process > 'Ascat (P1T_vs_P1N)'

Caused by:
  No signature of method:
    for f in *BAF *LogR; do sed 's/chr//g' $f > tmpFile; mv tmpFile $f;done
    run_ascat.r --tumorbaf P1T.BAF --tumorlogr P1T.LogR --normalbaf P1N.BAF --normallogr P1N.LogR --tumorname P1T --basedir /home/ec2-user/.nextflow/assets/ggabernet/nf-core-sarek --gcfile 1000G_phase3_GRCh38_maf0.3.loci.gc --gender XY
    () is applicable for argument types: (String) values: []

Source block:
  gender = genderMap[idPatient]
  ascat_purity=params.ascat_purity
  ascat_ploidy=params.ascat_ploidy
  purity_ploidy = (params.ascat_purity && params.ascat_ploidy) ? "--purity ${params.ascat_purity} --ploidy ${params.ascat_ploidy}" : ""
  """
      for f in *BAF *LogR; do sed 's/chr//g' \$f > tmpFile; mv tmpFile \$f;done
      run_ascat.r --tumorbaf ${bafTumor} --tumorlogr ${logrTumor} --normalbaf ${bafNormal} --normallogr ${logrNormal} --tumorname ${idSampleTumor} --basedir ${baseDir} --gcfile ${acLociGC} --gender ${gender} ${purity_ploidy}
      """""

Tip: you can replicate the issue by changing to the process work dir and entering the command `bash .command.run`
Unexpected error [AbortedException]

 -- Check script '/home/ec2-user/.nextflow/assets/ggabernet/nf-core-sarek/main.nf' at line: 832 or see '.nextflow.log' file for more details
```

Reply:
Remove that, I'll improve that in a separate PR then.
Thanks for your help